torch_xla/csrc/runtime/pjrt_computation_client.h

```diff
@@ -175,6 +175,14 @@ class PjRtComputationClient : public ComputationClient {
       return buffer != nullptr && !buffer->IsDeleted();
     };
 
+    bool HasSharding() const override { return false; }
```

Comment:
Is this duplicated with definition below?

Reply:
haha no, this is for PjrtData and below is PjrtShardedData